frontend/src/static/css/style.scss

```diff
@@ -152,6 +152,11 @@ header{
     padding:2rem 1rem;
 }
 
+.contribution-text-font {
```

Comment:
`.contribution-text-font` should be nested within `.summary-charts`

Reply:
actually, instead of `contribution-text-font`, we can just give it a representative name e.g. `mini-font` and make it a global standard in our style sheet.